integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -176,6 +176,8 @@ private int createOrOpenInternal(String path, FuseFileInfo fi, long mode) {
       return res;
     }
     try {
+      mFileSystem.createDirectory(uri.getParent(),
+          CreateDirectoryPOptions.newBuilder().setRecursive(true).build());
```

Comment:
why is this necessary?

Reply:
Thank you, but this change is removed.

Actually, because if the dir is a multiple level dir, then create file will fail because its parent dir does not exist.

Reply:
Let me explain more here, because in latest patch, this is still needed.

In our tests, we may create some files in advance, write some data into these files.
Then tests may read from these files.
These files are created/filled in UFS. In UfsBaseFileSystem, the input path will be treaded absolute/full path in UFS.
But the file path passed to other layer of file system is Alluxio based path.

If the file path is interpreted as Alluxio path, then some parent directories maybe don't exist.
We need to create those directories first.